docs/model_configuration.md

```diff
@@ -454,16 +454,45 @@ value. For the above example, an inference request must provide inputs
 with the following shapes.
 
 ```
-  "input0": [ x, -1]
-  "input1": [ 1 ]
-  "output0": [ x, -1]
+  "input0": [ x, 1, 3]
+  "input1": [ 3 ]
```

Comment:
Why `input1` has shape [3] instead of [2] (config)

Reply:
Meaning config shape + 1 (if batching)?

Reply:
That is because of the batching. Let me try to elaborate on that.